app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -5,6 +5,8 @@ module MiqPolicyController::Alerts
     helper_method :display_driving_event?
   end
 
+  SEVERITIES = {"info" => _('Info'), "warning" => _('Warning'), "error" => _('Error')}.freeze
```

Comment:
gettext is not going to work with a constant like this (constant is loaded & evaluated only once). You have to use `N_()` here and `_()` in the place where you actually use the constant.

Reply:
@mzazrivec Fixed.